### ISSUE TITLE
warning outputs garbled when -j 6 option

### ISSUE THREAD

**Issue by bdbaddog** [SCons/scons#1746]
This issue was originally created at: 2007-10-07 00:22:54.
This issue was reported by: `issues@scons`.
>issues@scons said at 2007-10-07 00:22:54

Building static libraries with `-j 6 -s options` on quad opteron using msvc 2008 beta 2, python 2.5.1.  When the compiler generates warnings on files built in parallel, the warning messages displayed in the command window become garbled/mixed.  Without `-j 6`, no problem.

>gregnoel said at 2008-05-30 12:59:34

Bug party triage.
 
*** This issue has been marked as a duplicate of 1183 ***

>gregnoel said this issue  is a duplicate of #1183 at 2008-05-30 12:59:34.
